sdk/search/search-documents/package.json

```diff
@@ -105,7 +105,7 @@
     "eslint": "^9.9.0",
     "playwright": "^1.49.0",
     "type-plus": "^7.6.2",
-    "typescript": "~5.3.3",
+    "typescript": "~5.7.2",
```

Comment:
compiler error gone? https://github.com/Azure/azure-sdk-for-js/issues/29283

Reply:
ah interesting. I reverted it in https://github.com/Azure/azure-sdk-for-js/pull/32160/commits/412cb399a9106bff49b42145acd393096c9e5387